sonar-batch/src/main/java/org/sonar/batch/test/DefaultTestCase.java

```diff
@@ -70,100 +70,95 @@ public MutableTestCase setDurationInMs(@Nullable Long l) {
     if (l != null && l < 0) {
       throw new IllegalDurationException("Test duration must be positive (got: " + l + ")");
     }
-    setProperty(DURATION, l);
+    this.durationInMs = l;
     return this;
   }
 
   @Override
   public Status status() {
-    return Status.of((String) getProperty(STATUS));
+    return status;
   }
 
   @Override
   public MutableTestCase setStatus(@Nullable Status s) {
-    setProperty(STATUS, s == null ? null : s.name());
+    this.status = s;
+    ;
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png)Remove this empty statement.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)